pkgs/development/compilers/dotnet/combine-packages.nix

```diff
@@ -32,14 +33,13 @@ mkWrapper "sdk" (buildEnv {
     "templates"
   ];
   ignoreCollisions = true;
-  nativeBuildInputs = [ makeWrapper ];
   postBuild =
     ''
       mkdir -p "$out"/share/dotnet
-      cp "${cli}"/share/dotnet/dotnet $out/share/dotnet
+      cp "${cli}"/share/dotnet/${executableName} $out/share/dotnet
```

Comment:
Shouldn't the target use `executableName` as well?

Reply:
It does, it copies `share/dotnet/dotnet[.exe]` into `share/dotnet` (a directory)

Reply:
I see, forget about it then